ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -788,8 +765,6 @@ public void onCompletion(ReadableStreamChannel routerResult, Exception routerExc
     public void onCompletion(String routerResult, Exception routerException) {
       callbackTracker.markOperationEnd();
       if (routerResult == null && routerException == null) {
-        frontendMetrics.routerCallbackError.inc();
-        callbackTracker.markCallbackProcessingEnd();
```

Comment:
Similar to my previous comment about this being a programming error and metrics not being of much use. What do you think?

Processing end is marked in `finally` in any other case.